g.sh

```
@@ -1,6 +1,6 @@
 import time
 import json
-from bitcoin import *
+from bitcoin import
 
 # BlockCypher API Token
 api_token = "8bd4fa2488614e509a677103b88b95fc"
```

Shellcheck (0.10.0)

[error] Remove spaces around = to assign (or use [ ] to compare, or quote '=' if literal).

(SC2283)